core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/DummyTextAnnotationGenerator.java

```diff
@@ -195,113 +212,151 @@ public static TextAnnotation generateBasicTextAnnotation(int numSentences) {
 
     private static String[] allPossibleViews = new String[] {ViewNames.POS, ViewNames.LEMMA,
             ViewNames.SHALLOW_PARSE, ViewNames.PARSE_GOLD, ViewNames.SRL_VERB, ViewNames.NER_CONLL,
-            ViewNames.PSEUDO_PARSE_STANFORD };
+            ViewNames.PSEUDO_PARSE_STANFORD};
 
-    public static TextAnnotation generateAnnotatedTextAnnotation(boolean withNoisyLabels) {
-        return generateAnnotatedTextAnnotation(allPossibleViews, withNoisyLabels);
+    public static TextAnnotation generateAnnotatedTextAnnotation(boolean withNoisyLabels,
+                                                                 int sentenceLength) {
+        return generateAnnotatedTextAnnotation(allPossibleViews, withNoisyLabels, sentenceLength);
     }
 
     public static TextAnnotation generateAnnotatedTextAnnotation(String[] viewsToAdd,
-            boolean withNoisyLabels) {
-        TextAnnotation ta = BasicTextAnnotationBuilder.createTextAnnotationFromTokens(annotatedTokenizedStringArray);
+                                                                 boolean withNoisyLabels, int sentenceLength) {
+        List<String[]> annotatedTokenizedStringArrayAll = new ArrayList<>();
+        annotatedTokenizedStringArrayAll.addAll(annotatedTokenizedStringArray1);
+        if(sentenceLength > 1)
+            annotatedTokenizedStringArrayAll.addAll(annotatedTokenizedStringArray2);
+        if(sentenceLength > 2)
```

Comment:
If the input is 4, the user will still get 3 sentences back. That's a bit misleading. Maybe throw a warning (or exception)?